packer/build_image.sh

```diff
@@ -224,6 +224,40 @@ elif [ "$TARGET" = "azure" ]; then
     SSH_USERNAME=azureuser
     SCYLLA_IMAGE_DESCRIPTION="scylla-$SCYLLA_FULL_VERSION scylla-machine-image-$SCYLLA_MACHINE_IMAGE_VERSION scylla-python3-$SCYLLA_FULL_VERSION"
 
+    RESOURCE_GROUP="SCYLLA-IMAGES"
+    GALLERY_NAME="scylladb_dev"
+    LOCATION="eastus"
+    DEF_NAME=$BRANCH
+
+    az sig create \
+      --resource-group "$RESOURCE_GROUP" \
+      --gallery-name "$GALLERY_NAME" \
+      --location "$LOCATION" \
+      --output none 2>/dev/null || echo "   -> Gallery '$GALLERY_NAME' already exists (or created)."
+
+    az sig image-definition create \
+      --resource-group "$RESOURCE_GROUP" \
+      --gallery-name "$GALLERY_NAME" \
+      --gallery-image-definition "$DEF_NAME" \
+      --publisher "ScyllaDB" \
+      --offer "scylla" \
+      --sku "$DEF_NAME" \
+      --os-type Linux \
+      --os-state Generalized \
+      --hyper-v-generation V2 \
+      --features "DiskControllerTypes=SCSI,NVMe" \
+      --location "$LOCATION" \
+      --output none || echo "   -> Image definition '$DEF_NAME' already exists (or created)."
```

Comment:
Both should happen only once per release, why doing it for every image creation ?